frontend-addon/inputHandler.ts

```diff
@@ -4,7 +4,7 @@ function onEventNameChange(e) {
 }
 
 function isEmailValid(email: string): boolean {
-    return email.endsWith("@gmail.com");
+    return email.endsWith("@gmail.com") || email.endsWith("@student.pwr.edu.pl");
```

Comment:
id add here also email ending with @pwr.edu.pl in case they'd want to run the  app